feature-group-definitions/stable-array-sort.yml

```diff
@@ -0,0 +1,16 @@
+name: Stable array sort
```

Comment:
Array.prototype.sort() was introduced in ECMAScript 1 and the stable version of the method was only specified in ECMAScript 2019. I thought it would therefore either deserve its own feature or, as you say, be ignored completely. How do we assess relevance of BCD sub features?

Reply:
OK, so going only by the Baseline date this feature is absolutely still relevant.

I have left BCD subfeatures unmapped when I judged them to be "small" problems, but haven't thought through principles. I think that lack of stable sort is a "small" problem because ~everyone got by fine for decades before it was made consistent across browsers.

This BCD entry doesn't have its own MDN page, so it's not important to map for Baseline badging purposes. But I don't have a principled argument here. Can we let @ddbeck decide? :)

Reply:
I thought sort() is probably one of the very relevant/heavily used methods of arrays, so changes to it seem important to me, but I have no additional data points on this feeling.

Caniuse surfaces it through BCD import: https://caniuse.com/mdn-javascript_builtins_array_sort_stable_sorting
Not sure how to mark it, so that caniuse would show the baseline banner there. Should I add `caniuse: mdn-javascript_builtins_array_sort_stable_sorting`?

Reply:
So let's keep this entry. I think no caniuse link when it's imported from BCD though, that should be possible to match up with code.

Reply:
> Should I add `caniuse: mdn-javascript_builtins_array_sort_stable_sorting`?

> I think no caniuse link when it's imported from BCD though, that should be possible to match up with code.

This _shouldn't_ work, which is correct, IMO. `caniuse-lite` only gives us the features that come from caniuse directly (with a few odd exceptions), not the features the MDN shows.